src/main/java/com/intel/gkl/pairhmm/IntelPairHmm.java

```diff
@@ -68,10 +67,33 @@ public void initialize(PairHMMNativeArguments args) {
             args.useDoublePrecision = false;
             args.maxNumberOfThreads = 1;
         }
+
         if (args.useDoublePrecision && useFpga) {
             logger.warn("FPGA PairHMM does not support double precision floating-point. Using AVX PairHMM");
         }
+
+        if(!gklUtils.getFlushToZero()) {
+            logger.warn("Flush-to-zero (FTZ) is disabled; enabling just for PairHMM");
+        }
```

Comment:
Actually, I added code that disables FTZ in the ```done()``` function [here](https://github.com/Intel-HLS/GKL/blob/847f6f8915952559245c17ebd686d5f5e18faf14/src/main/native/pairhmm/IntelPairHmm.cc#L144), only if it was disabled before ```init()``` was first called. That is the reason behind all these messages.

Should I get rid of that code/logic? Do we want to leave FTZ enabled after PairHMM runs _even if it was disabled to begin with_?